libopenage/renderer/opengl/renderer.cpp

```diff
@@ -169,6 +171,9 @@ void GlRenderer::render(const std::shared_ptr<RenderPass> &pass) {
 	auto gl_target = std::dynamic_pointer_cast<GlRenderTarget>(pass->get_target());
 	gl_target->bind_write();
 
+	// bind the shared quad vao for all bufferless quad geometries
```

Comment:
I slightly reworded the comment so that it's more clear what's happening.

```suggestion
	// ensure that an (empty) VAO is bound before rendering geometry
	// a bound VAO is required to render bufferless quad geometries by OpenGL
	// see https://www.khronos.org/opengl/wiki/Vertex_Rendering#Causes_of_rendering_failure
```